packages/svelte-ux/src/lib/components/ThemeSwitch.svelte

```diff
@@ -4,9 +4,18 @@
   import Switch from './Switch.svelte';
   import Icon from './Icon.svelte';
 
-  import { getSettings } from './settings.js';
+  import { getComponentSettings, getSettings } from './settings.js';
+  import { cls } from '../utils/index.js';
+  import type { ComponentProps } from 'svelte';
+  import { clsMerge } from '$lib/utils/styles.js';
 
   const { currentTheme } = getSettings();
+
+  const { classes: settingsClasses } = getComponentSettings('ThemeSwitch');
+
+  export let classes: {
+    icon?: string;
+  } & ComponentProps<Switch>['classes'] = {};
```

Comment:
`classes.root` would actually be passed down as the `root` for the underlying `Switch` which might be slightly confusing but I think it's fine

Reply:
This is interesting as I was initially expecting adding `switch` to `classes` here, but `classes.switch.switch` is a little odd so handling it this way seems fine (just different than normal)

Reply:
I figured we didn't need to expose the details of the component composition via the prop api, although maybe that's easier to understand 🤷‍♂️